### ISSUE TITLE
Avatar - Uploading/changing image cannot be accessed with a keyboard

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#718]
### Expected behavior
- Image or button can receive focus for uploading function. 
- Can use keyboard to choose a new file for the image.
### Actual behavior

There is no way to access this function using the keyboard.
### Steps to reproduce
1. Tab through page and try to land on the canvas or button for uploading/changing image
2. No element is focusable for this

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.0](https://github.com/blackbaud/skyux/releases/tag/v1.7.0)
